diff --git a/dist/internal/vite-plugins.js b/dist/internal/vite-plugins.js
index c6be5cc8713bfa484271f927be03e6014042f207..2e60acea95eafb54d945fbfd08dea6226dbdaa1a 100644
--- a/dist/internal/vite-plugins.js
+++ b/dist/internal/vite-plugins.js
@@ -67,6 +67,8 @@ export function deps() {
                         'vocs > lz-string',
                         'vocs > react-error-boundary',
                         'vocs > ts-interface-checker',
+                        'vocs > mermaid',
+                        'vocs > mermaid > dayjs',
                         ...(config?.optimizeDeps?.include ?? []),
                     ],
                     exclude: [
